incubator/astro/templates/NOTES.txt

```diff
@@ -0,0 +1,3 @@
+Thanks for installing astro! To see what it's doing, use `kubectl logs` for your new astro pod.
```

Comment:
Would be nice to have some instructions on how to find that pod. Maybe just a simple:

```
kubectl get po -n {{ .Release.Namespace }}
```

If you want to get more detailed, you could save the pod name in a variable and provide a logs command.